src/components/Panel/_mixins.scss

```diff
@@ -10,13 +10,13 @@
 @import '../../components/IconButton/mixins';
 @import '../../components/Toolbar/mixins';
 
-@mixin panel {
-  /// Width.
-  /// @type Length
-  $panel__sizing__width: carbon--mini-units(
-    $count: 40,
-  );
+/// Width.
+/// @type Length
+$panel__sizing__width: carbon--mini-units(
```

Comment:
Needed to move this out so it can be used by `PanelV2`. 
(NOTE: we should probably move all of these `Panel` variables up in scope like other components? But for this PR I wanted to keep changes at a minimum so I only moved the variable that was necessary 👍 )